### PR TITLE
Nggw 106 vscode settings

### PR DESCRIPTION
**Pull request by markosandalj**
As discussed with @iherak I added basic editor settings for vs code where we defined tabSize and file end of line for frequently used file types. I also added file for recommended vs code extensions that can later be updated if needed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,7 +35,6 @@ yarn-error.log
 ###< liip/imagine-bundle ###
 
 .idea/
-.vscode/
```
Comment (emodric) [inline .gitignore:38]: This needs to be setup differently (e.g. like for `bin` folder below).

The point is to allow the two files you added, but disallow everything else, e.g:

```
.vscode/*
!.vscode/extensions.json
!.vscode/settings.json
```

File: .vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "mblode.twig-language-2",
+    "kd132.netgen-snippets"
+  ]
+}
```
Comment (emodric) [inline .vscode/extensions.json:6]: Missing new line @ EOF

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    "[php]": {
+        "editor.tabSize": 4,
+        "editor.detectIndentation": false,
+    },
+    "[yaml]": {
+        "editor.tabSize": 4,
+        "editor.detectIndentation": false,
+    },
+}
```
Comment (emodric) [inline .vscode/settings.json:27]: Missing new line @ EOF

**Comment by emodric** [comment]
Thanks for ŧhis @markosandalj !

You can target `1.12` branch with this too, no need to go only on `master`.

File: .gitignore (hunk 1 above)
Comment (markosandalj) [reply .gitignore:38]: This should be done now.

**Comment by emodric** [comment]
Thanks @markosandalj !

**Pull request merged**